.github/workflows/cpp-tests-windows.yml

```
@@ -0,0 +1,79 @@
+
+      - name: Setup ccache
+        uses: hendrikmuhs/ccache-action@5ebbd400eff9e74630f759d94ddd7b6c26299639 # v1.2
+        with:
+          key: "${{ inputs.runs-on }}-${{ inputs.toolset }}-Release"
+          variant: ccache
+          # Only `main` writes; pull requests restore its cache.
+          save: ${{ github.ref == 'refs/heads/main' }}
+          max-size: 10G
+
+      - name: Create Build Environment
+        run: cmake -E make_directory ${{ github.workspace }}\build_release
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC1001:info:1:48: This \b will be a regular 'b' in this context

(shellcheck)